lib/Viroverse/ISLAWorksheet.pm

```diff
@@ -0,0 +1,260 @@
+use strict;
+use warnings;
+use utf8;
+
+package Viroverse::ISLAWorksheet;
+use Moo;
+use Types::Standard qw< :types >;
+use Viroverse::Logger qw< :log >;
+
+=head1 NAME
+
+Viroverse::ISLAWorksheet - convert between a Viroverse::Result::Sample and an
```

Comment:
Be careful with the names of things: the class name is `ViroDB::Result::Sample`. Also consider adding some light POD markup, e.g. `L<ViroDB::Result::Sample>` will create a link in the right circumstances. (I've long wanted to do a nice HTML dump of the POD from Viroverse because there's enough code in here that'd be nice to actually browse the documentation, but I've never been quite happy with the tooling I've found for this.)